src/charm.lua

```diff
@@ -1191,13 +1229,78 @@ if next(SMODS.find_mod 'Cryptid') then
     local raw_get_weight = SMODS.Rarities.Common.get_weight
     SMODS.Rarity:take_ownership("Common", {
         get_weight = function(...)
-            return G.GAME and G.GAME.Bakery_charm == 'BakeryCharm_Bakery_DuctTape' and 0 or raw_get_weight(...)
+            return G.GAME and G.GAME.Bakery_charm == 'BakeryCharm_Bakery_DuctTape'
+                and not is_charm_debuffed_safe() and 0 or raw_get_weight(...)
         end
     })
     local raw_get_weight = SMODS.Rarities.Common.get_weight
     SMODS.Rarity:take_ownership("Uncommon", {
         get_weight = function(...)
-            return G.GAME and G.GAME.Bakery_charm == 'BakeryCharm_Bakery_DuctTape' and 0 or raw_get_weight(...)
+            return G.GAME and G.GAME.Bakery_charm == 'BakeryCharm_Bakery_DuctTape'
+                and not is_charm_debuffed_safe() and 0 or raw_get_weight(...)
```

Comment:
The check on this is funny to me, since it applies to Judgement and basically nothing else.